views/partials/email_partials/email_button.hbs

```diff
@@ -5,8 +5,12 @@
         <table>
           <tr>
             <td class="email-button-wrapper" align="center" bgcolor="#0060df" style="width: 100%; max-width: 530px; border-radius: 2px;">
-              <a class="email-button" href="{{ href }}" rel="noopener" style="text-align: center; text-decoration: none; color: #ffffff; padding-top: 12px; padding-right: 30px; padding-left: 30px; padding-bottom: 12px; border: 1px solid #0060df; display: inline-block; font-size: 13px;">
-               {{ buttonValue }}
+              <a class="email-button" href="{{ scanAnotherEmailHref }}{{ verificationHref }}" rel="noopener" style="text-align: center; text-decoration: none; color: #ffffff; padding-top: 12px; padding-right: 30px; padding-left: 30px; padding-bottom: 12px; border: 1px solid #0060df; display: inline-block; font-size: 13px;">
```

Comment:
Whoa, this is a crazy trick that works for now, but it "smells" like it may not be future-proof. I.e., if we add an email that contains both `verificationHref` and `scanAnotherEmailHref`, they will conflict in here.

Reply:
Very good point. Did some moving around to fix this.